test/common.js

```diff
@@ -291,29 +291,25 @@ module.exports = (endpoint, mandatory, optional = [], associations = []) => {
             postError(done, body, 400);
           });
 
-          xit('should respond 201 if request to link ' + endpoint +
+          it('should respond 201 if request to link ' + endpoint +
               ' to existing ' + name + ' is valid', done => {
             db().then(models => {
               models[name].create(CONST[name + 'Entity']).then(relation => {
                 let body = Object.assign({}, testEntity);
```

Comment:
This test is not correct. The body in this case should only contain `{ '@iot.id': relation.id }`